tests/markers.py

```diff
@@ -1,12 +1,13 @@
 import enum
 import os
-from typing import Set, Type
-from unittest import TestCase
+from typing import Set, TypeVar
 
 _MARKERS: Set[str] = {
     marker.strip() for marker in os.getenv("DISABLED_TESTS", "").split(",")
 }
 
+T = TypeVar("T")
```

Comment:
I think that this type variable should be bound to `TestCase` since it is used in a context where we would only ever want descendants of `TestCase` to be passed in.